app/src/main/java/swati4star/createpdf/fragment/MergeFilesFragment.java

```diff
@@ -0,0 +1,311 @@
+import com.itextpdf.text.pdf.PdfImportedPage;
+import com.itextpdf.text.pdf.PdfReader;
+import com.itextpdf.text.pdf.PdfWriter;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.Objects;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import swati4star.createpdf.R;
+import swati4star.createpdf.util.StringUtils;
+
+import static android.app.Activity.RESULT_OK;
+
+
+public class MergeFilesFragment extends Fragment {
+    private Activity mActivity;
+    private final Context mContext;
+    private String mPath;
+    public String val1, val2;
+    boolean success;
+    String mFilename;
+    public String btTag = "";
+    public static final int INTENT_REQUEST_PICKFILE_CODE = 10;
+    String ret;
+    @BindView(R.id.textView)
+    TextView nosupport;
+    @BindView(R.id.fileonebtn)
+    Button addFileOne;
+    @BindView(R.id.filetwobtn)
+    Button addFileTwo;
+    @BindView(R.id.mergebtn)
+    Button mergeBtn;
+    @BindView(R.id.txtfirstpdf)
+    EditText txt1;
+    @BindView(R.id.txtsecondpdf)
+    EditText txt2;
+
+
+    public MergeFilesFragment() {
+        // Required empty public constructor
+        this.mContext = this.getContext();
+    }
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+
+
+        View root = inflater.inflate(R.layout.fragment_merge_files, container, false);
+        ButterKnife.bind(this, root);
+        return root;
+    }
+
+    @OnClick(R.id.fileonebtn)
+    void startAddingPDF(View v) {
+        Log.d("img", "startAddingPDF: ");
+        btTag = (v).getTag().toString();
+        showFileChooser();
+    }
+
+    @OnClick(R.id.filetwobtn)
+    void startAddingPDF2(View v) {
+        Log.d("img", "startAddingPDF: ");
+        btTag = (v).getTag().toString();
+        showFileChooser();
+    }
+
+    @OnClick(R.id.mergebtn)
+    void mergeFiles(final View v) {
+        new MaterialDialog.Builder(mActivity)
+                .title(R.string.creating_pdf)
+                .content(R.string.enter_file_name)
+                .input(getString(R.string.example), null, new MaterialDialog.InputCallback() {
+                    @Override
+                    public void onInput(@NonNull MaterialDialog dialog, CharSequence input) {
+                        if (StringUtils.isEmpty(input)) {
+                            Snackbar.make(Objects.requireNonNull(mActivity).findViewById(android.R.id.content),
+                                    R.string.snackbar_name_not_blank,
+                                    Snackbar.LENGTH_LONG).show();
+                        } else {
+                            mFilename = input.toString();
+                            mergePdfFiles(v);
+                        }
+                    }
+                })
+                .show();
+    }
+
+    public void mergePdfFiles(View view) {
```

Comment:
You don't need view parameter here

Reply:
I've used it to refer and find which button was being clicked as I saw this was a way they were referring to online. I can remove it and check as well.